src/main/java/org/fundacion/jala/converter/models/facade/ZipFileFacade.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Zacarias Huanca Alconz
+ */
+package org.fundacion.jala.converter.models.facade;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import static org.fundacion.jala.converter.models.AssetSQL.insertAssetData;
+import static org.fundacion.jala.converter.service.ZipService.zipFile;
+import static org.fundacion.jala.converter.service.ZipService.zipFiles;
+
+public class ZipFileFacade {
+
+    private ZipFileFacade() {
+    }
+
+    /**
+     * Compress files of Audio
+     *
+     * @param parameterOutputChecksum object of ParameterOutputChecksum
+     * @param metadata if add metadata of audio into zip
+     * @param storagePath path and name of file to compress
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    public static void getZipFileAudio(final ParameterOutputChecksum parameterOutputChecksum,final String metadata,
+                                       final String storagePath) throws IOException, InterruptedException {
+        getZipFile(parameterOutputChecksum, metadata, storagePath, false);
+    }
+
+    /**
+     * Compress files of video
+     *
+     * @param parameterOutputChecksum object of ParameterOutputChecksum
+     * @param metadata if add metadata of video into zip
+     * @param storagePath path and name of file to compress
+     * @param thumbnail if add thumbnail of video into zip
+     * @throws IOException when invalid path is given in zipFiles
+     * @throws InterruptedException is exception if process is interrupted
+     */
+    public static void getZipFileVideo(final ParameterOutputChecksum parameterOutputChecksum,final String metadata,
+                                       final boolean thumbnail,final String storagePath) throws IOException, InterruptedException {
+        getZipFile(parameterOutputChecksum, metadata, storagePath, thumbnail);
+    }
+
+    /**
+     * Compress files
+     *
+     * @param parameterOutputChecksum object of ParameterOutputChecksum
+     * @param metadata if add metadata into zip
+     * @param storagePath path and name of file to compress
+     * @param thumbnail if add thumbnail of video into zip
+     * @throws IOException when invalid path is given in zipFiles
+     * @throws InterruptedException is exception if process is interrupted
+     */
+    private static void getZipFile(final ParameterOutputChecksum parameterOutputChecksum,final String metadata,
```

Comment:
A space is required between the parameters and the ","

Reply:
ok